crates/packager/src/cli/mod.rs

```diff
@@ -285,6 +285,9 @@ where
 
     if !cli.quite {
         init_tracing_subscriber(cli.verbose);
+        if std::env::var_os("CARGO_TERM_COLOR").is_none() {
+            std::env::set_var("CARGO_TERM_COLOR", "always");
+        }
```

Comment:
shouldn't this be set by the user invoking the packager? or at least kept as `auto` so it automatically detects whether the current terminal has color support or not?

Reply:
if we do not force this there's no colored output at all :(